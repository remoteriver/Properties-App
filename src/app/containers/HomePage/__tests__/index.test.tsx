import * as React from 'react';
import { Provider } from 'react-redux';
import { createRenderer } from 'react-test-renderer/shallow';
import { configureAppStore } from 'store/configureStore';
import { HomePage } from '../index';

const renderer = createRenderer();

describe('<HomePage />', () => {
  let store;

  beforeAll(() => {
    store = configureAppStore();
  });

  it('should render and match the snapshot', () => {
    renderer.render(
      <Provider store={store}>
        <HomePage />
      </Provider>,
    );
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
