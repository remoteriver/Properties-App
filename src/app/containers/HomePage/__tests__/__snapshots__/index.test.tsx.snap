// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render and match the snapshot 1`] = `
<Context.Provider
  value={
    Object {
      "store": Object {
        "createReducer": [Function],
        "dispatch": [Function],
        "getState": [Function],
        "injectedReducers": Object {},
        "injectedSagas": Object {},
        "replaceReducer": [Function],
        "runSaga": [Function],
        "subscribe": [Function],
        Symbol(Symbol.observable): [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "notify": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "createReducer": [Function],
          "dispatch": [Function],
          "getState": [Function],
          "injectedReducers": Object {},
          "injectedSagas": Object {},
          "replaceReducer": [Function],
          "runSaga": [Function],
          "subscribe": [Function],
          Symbol(Symbol.observable): [Function],
        },
        "unsubscribe": null,
      },
    }
  }
>
  <HomePage />
</Context.Provider>
`;
